DESCRIPTION

```diff
@@ -36,55 +36,55 @@ Imports:
     methods,
     S4Vectors,
     crayon,
-    Matrix
+    Matrix,
+    pkgconfig,
+    AnnotationDbi,
+    DESeq2,
+    Rtsne,
+    Seurat,
+    betareg,
+    boot,
+    broom, 
+    class, 
+    clusterProfiler, 
+    e1071, 
+    edgeR,
+    functional, 
+    glmmSeq, 
+    glmmTMB, 
+    limma, 
+    lme4, 
+    matrixStats,
+    msigdbr, 
+    org.Hs.eg.db,
+    org.Mm.eg.db, 
+    pbapply, 
+    pbmcapply,
+    survival,
+    survminer, 
```

Comment:
Thanks for taking this through!

I really think the dependency structure should go back as before. Please try to remove dependency gradually and see if error appear.

Is there any reason to add tidySummarizedExperiment in Suggests?

**please open a new branch for this*